README.md

```diff
@@ -1,13 +1,19 @@
+<!---
+Logo
+-->
 <p align="center">
   <a href="https://docsify.js.org">
     <img alt="docsify" src="https://github.com/docsifyjs/docsify/blob/develop/docs/_media/icon.svg">
   </a>
 </p>
 
 <p align="center">
-  A magical documentation site generator.
+  <b>Docsify</b>, a magical documentation site generator that has no statically built html files. It is simple and lightweight with a smart full-text search plugin and multiple themes, useful plugin API, compatible with IE11. It also supports both <a href="https://github.com/docsifyjs/docsify-ssr-demo">SSR</a> and embedded files.
```

Comment:
@sy-records this is good, honestly we need more details because "magical" on the cover doesn't say much. Although I think we might become incompatible with IE11 now, and we shouldn't mention experimental and incomplete/broken SSR support.

Reply:
Isn't there a description of some Features below? https://docsify.js.org/#/?id=features